app/src/main/java/org/oppia/app/player/state/HtmlParserEntityTypeModule.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.app.player.state
+
+import dagger.Module
+import dagger.Provides
+
+/** Provides Html parsing entity type dependencies. */
+@Module
+class HtmlParserEntityTypeModule {
```

Comment:
Suggest putting this next to the HtmlParser in the same package.

Reply:
Done.